source/api-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _rm-rest-api:
+
+============================
+Relational Migrator REST API
+============================
+
+.. facet::
+   :name: genre 
+   :values: tutorial
+
+.. meta:: 
+   :description: Learn about the MongoDB Relational Migrator REST API and how to use it.
+   :keywords: data migration, rest api, open api
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The Relational Migrator REST API provides you with programmatic access 
+to Relation Migrator with HTTPS endpoints. The REST API is a middleware 
+service that sits between your application and your source and destination 
```

Comment:
Avoid "sit" (not global)
Also, use something other than "middleware" if you can.